src/web-ui/src/partials/TextAlerts/TextAlerts.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+  <section v-if="isEnabled && user" class="section container p-4">
+    <h1 class="heading mb-1">Join <span class="text-alerts">text alerts</span> and get 20% off</h1>
+    <p class="disclaimer mb-2">Message and data rates may apply. See details.</p>
+    <form @submit.prevent="onSubmit" class="form d-flex justify-content-center align-items-stretch">
+      <TheMask
+        type="tel"
+        name="phoneNumber"
+        placeholder="Enter cellphone number"
+        v-model="phoneNumber"
+        mask="+1 (###) ### - ####"
+        class="input py-1 px-2"
+      />
+      <button type="submit" class="submit btn">Submit</button>
+    </form>
+  </section>
+</template>
+
+<script>
+import { mapActions, mapState } from 'vuex';
+import { TheMask } from 'vue-the-mask';
+import { RepositoryFactory } from '@/repositories/RepositoryFactory';
+
+const UsersRepository = RepositoryFactory.get('users');
+
+export default {
+  name: 'TextAlerts',
+    components: {
+      TheMask
+    },
+  data() {
+    return {
+      isEnabled: process.env.VUE_APP_PINPOINT_APP_ID ,
```

Comment:
should we called this isPinpointEnabled?

Reply:
Done